lib/usage.js

```diff
@@ -122,9 +134,10 @@ module.exports = function (yargs, y18n) {
         return acc
       }, {})
     )
+    var theWrap = getWrap()
```

Comment:
Well, `maxWidth` is called but not _defined_ within `.help()`, that's why the extra arg would be necessary. But if we just moved it, then yes. Either way, I'm all for it.

Sorry to pick nits here. I really do appreciate the contribution!